fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/converters/PropertyConverter.java

```diff
@@ -94,7 +94,10 @@ public static String getPropertyNameFromPredicate(final Node node,
                                                       final Resource predicate,
                                                       final Map<String, String> namespaceMapping)
             throws RepositoryException {
-        final NamespaceRegistry namespaceRegistry = getNamespaceRegistry.apply(node);
+
+        final NamespaceRegistry namespaceRegistry = uncheck((final Node n) ->
```

Comment:
Was there are reason for not just using `NamespaceTools.getNamespaceRegistry(n)`?
https://github.com/fcrepo4/fcrepo4/pull/882/files#diff-a456c1fc2649a798904fe05c3ae2e5fdR46